homeassistant/components/blink/const.py

```diff
@@ -7,6 +7,7 @@
 CONF_MIGRATE = "migrate"
 CONF_CAMERA = "camera"
 CONF_ALARM_CONTROL_PANEL = "alarm_control_panel"
+CONF_DEVICE_ID = "device_id"
```

Comment:
haven't explciitly checked it, but do we use this?

Reply:
I take that back - it's used in the tests...